pkg/console/install_panels.go

```diff
@@ -1293,7 +1345,11 @@ func addNetworkPanel(c *Console) error {
 			if err != nil {
 				return "", err
 			}
-			mgmtNetwork.MTU, err = strconv.Atoi(mtu)
+
+			if mtu != "" {
+				mgmtNetwork.MTU, err = strconv.Atoi(mtu)
+			}
```

Comment:
Looks like if I type an "a" here and press up, it will still crash to the first panel.
Perhaps it's better to work like press enter (an error message is displayed).

Reply:
Additionally, the default value will be gone, not sure if this is a problem in my env.